frontend/src/libs/components/page-layout/page-layout.tsx

```diff
@@ -1,16 +1,16 @@
 import { type FC } from 'react';
 
+import { Header, RouterOutlet } from '~/libs/components/components.js';
```

Comment:
Let's import this from /header/header.js and router/router.js

Reply:
Should I add a PageLayout export from components.js? It wasn't exported there before and I figured I could import Header and RouterOutlet from there since there is no circular dependency.